config.go

```diff
@@ -689,7 +689,7 @@ type moduleConfig struct {
 // NewModuleConfig returns a ModuleConfig that can be used for configuring module instantiation.
 func NewModuleConfig() ModuleConfig {
 	return &moduleConfig{
-		startFunctions: []string{"_start"},
+		startFunctions: []string{"_start", "_initialize"},
```

Comment:
If both are present, would it be more correct to run _initialize first?

Reply:
I'd say so, yes.